src/ui/Panel.tsx

```diff
@@ -778,6 +778,18 @@ export class Panel extends Object2D {
             const coordinates = parts[1].split('-');
             this.setContig(contig);
 
+            if (!coordinates) {
+                throw new Error('Coordinates are invalid');
+            }
+
+            if (!coordinates[0]) {
+                throw new Error('First coordinate is invalid');
+            }
+
+            if (!coordinates[1]) {
+                throw new Error('Second coordinate is invalid');
+            }
+
```

Comment:
This code seems reasonable and I think mostly works as I'd expect.

(a) If I enter "chr1:11102837-", I get "Second coordinate is invalid."
(b) If I enter "chr1:-11267747", I get "First coordinate is invalid."
(c) If I enter "chr1", I get "Cannot read property split of undefined."
(d) If I enter "chr1:", I get "First coordinate is undefined."

I'm not sure what entry results in the "Coordinates are invalid" error, and I think the errors for (c) should be fixed.